cylc/flow/task_job_mgr.py

```diff
@@ -842,15 +874,20 @@ def _prep_submit_task_job_error(self, suite, itask, dry_run, action, exc):
 
     def _prep_submit_task_job_impl(self, suite, itask, rtconfig):
         """Helper for self._prep_submit_task_job."""
-        itask.task_owner = rtconfig['remote']['owner']
-        if itask.task_owner:
-            owner_at_host = itask.task_owner + "@" + itask.task_host
+        # itask.platform is going to get boring...
+        platform = itask.platform
+        host = get_host_from_platform(platform)
+        owner = platform['owner']
+
+        if owner:
+            owner_at_host = f"{owner}@{host}"
         else:
-            owner_at_host = itask.task_host
+            owner_at_host = host
+
         itask.summary['host'] = owner_at_host
```

Comment:
Will need to add upgrading the task summary thinggy to use platforms rather than hosts to the follow-on list.

Reply:
So I was under the impression that summary is info about the most recent instance of the task: It may turn out to be irrelevant, but I thought that the summary would want the more specific information.

Reply:
Nah, it doesn't matter what login node a job was submitted to, the UI etc should display the platform name.

Reply:
Yes, I see - it was the `itask.summary['job_hosts'][itask.submit_number]` which I was thinking should still explicitly record this, but I'm not sure about even that.

Reply:
There is also protobuf and graphql, we can chalk this up for laters.